src/middlewares/jsonBodyParser.js

```diff
@@ -3,8 +3,11 @@ const contentType = require('content-type')
 
 module.exports = () => ({
   before: (handler, next) => {
-    if (handler.event.headers && handler.event.headers['Content-Type']) {
-      const { type } = contentType.parse(handler.event.headers['Content-Type'])
+    const { headers } = handler.event
+    // normalize header
+    const headerContentType = headers['Content-Type'] || headers['content-type']
```

Comment:
you should probably check that `headers` is not undefined here as well

Reply:
Updated!